samtranslator/model/api/api_generator.py

```diff
@@ -610,13 +612,6 @@ def _add_cors(self):
         else:
             raise InvalidResourceException(self.logical_id, INVALID_ERROR)
 
-        if not SwaggerEditor.is_valid(self.definition_body):
-            raise InvalidResourceException(
-                self.logical_id,
-                "Unable to add Cors configuration because "
-                "'DefinitionBody' does not contain a valid Swagger definition.",
-            )
-
```

Comment:
removing all of these kind of validations, since they're now done when SwaggerEditor is instantiated